validator/remote/src/test/java/tech/pegasys/teku/validator/remote/eventsource/EventSourceBeaconChainEventAdapterTest.java

```diff
@@ -97,6 +100,45 @@ public void performsPrimaryReadinessCheckWhenFailoverNotReadyAndNoOtherFailovers
     verify(beaconNodeReadinessManager).performPrimaryReadinessCheck();
   }
 
+  @Test
+  public void dontJumpBetweenFailoversWhenFailoverIsReady() {
```

Comment:
Maybe rename it to `doNotSwitchToFailoverWhenCurrentBeaconNodeIsReady`

Reply:
Nice name, updated